src/main/java/MainApp.java

```diff
@@ -19,13 +29,64 @@ public void setUp() {
         options.addArguments("disable-infobars");
         driver = new ChromeDriver(options);
         wait = new WebDriverWait(driver, 60);
+
+        signInPage = new SignInPage(driver);
+        header = new Header(driver);
+        indexPage = new IndexPage(driver);
+        signInPage = new SignInPage(driver);
+        header = new Header(driver);
+        layerCard = new LayerCard(driver);
+        shoppingCard = new ShoppingCard(driver);
     }
 
     @AfterClass
     public void tearDown() {
         driver.quit();
     }
 
+    @Test
+    public void loginErrorTest() {
+        driver.get("http://automationpractice.com/index.php");
+        header.clickLoginLink();
+        signInPage.clickSignInLink();
+        assertThat(signInPage.isAlertOn());
+    }
+
+    @Test
+    public void TotalPriceTest() {
+        driver.get("http://automationpractice.com/index.php");
+        driver.manage().window().maximize();
+        Actions builder = new Actions(driver);
+        builder.moveToElement(indexPage.findElement().get(2)).build().perform();
+
+        indexPage.clickAddButton();
+
+        wait.until(ExpectedConditions.visibilityOf(layerCard.exitWindow()));
+        assertThat(layerCard.exitWindow().isDisplayed()).isTrue();
+        layerCard.exitWindowSuccess();
+
+        builder.moveToElement(indexPage.findElement().get(3)).build().perform();
+
+        indexPage.clickAddButton();
+        try {
+            Thread.sleep(2000);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        }
+        wait.until(ExpectedConditions.visibilityOf(layerCard.exitWindow()));
+        assertThat(layerCard.exitWindow().isDisplayed()).isTrue();
+        layerCard.exitWindowSuccess();
+
+        builder.moveToElement(header.moveToShopingCard()).build().perform();
+        wait.until(ExpectedConditions.visibilityOf(header.checkPriceCard()));
+        assertThat(header.checkPriceCard().isDisplayed()).isTrue();
+        header.openShoppingCard();
+
```

Comment:
this formatting should go to the page object, you can return formatted total and sum from page object

Reply:
done